Lib/asyncio/timeouts.py

```diff
@@ -51,7 +52,9 @@ def reschedule(self, when: Optional[float]) -> None:
         if when is None:
             self._timeout_handler = None
         else:
-            loop = events.get_running_loop()
+            # two asyncio event loops could be running on the same thread
+            # with different timers eg aioloop-proxy or interleaved asyncio.Runner
```

Comment:
```suggestion
            # we can't use get_running_loop() here because two event loops
            # could be running on the same thread with different timers eg
            # aioloop-proxy or interleaved asyncio.Runner
```